### PR TITLE
Fix typo

### PR DESCRIPTION
**Pull request by merwok**
Probably a mistake caused by “bekommen” :‑)

### CODE REVIEW

File: jinja2/loaders.py [hunk 1]
```diff
@@ -291,7 +291,7 @@ def list_templates(self):
 
 class FunctionLoader(BaseLoader):
     """A loader that is passed a function which does the loading.  The
-    function becomes the name of the template passed and has to return either
+    function receives the name of the template passed and has to return either
```
Comment (untitaker) [inline jinja2/loaders.py]: "recieves... passed" still sounds kind of weird.

Comment (merwok) [inline jinja2/loaders.py]: Right.  Fixed.

**Comment by untitaker** [comment]
Thanks!

**Comment by merwok** [comment]
Thanks!  I see “This pull request is closed, but the merwok:patch-1 branch has unmerged commits.”  I guess I can ignore that?  I see both of my changes in your commit.

**Comment by untitaker** [comment]
I've melded your commits into a single one under your name, so the ones in your branch are technically unmerged. You can ignore that msg by github

**Pull request closed without merging**